easy-kernel-workflow.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Set required variables
+EASY_KERNEL_WORKFLOW=${EASY_KERNEL_WORKFLOW:-"easy-kernel-workflow"}
+src_script_path=${src_script_path:-"$HOME/.config/$EASY_KERNEL_WORKFLOW/src"}
+
+set -e
```

Comment:
Definitely, we don't want this option here. It has to be removed because it makes hard to find errors and also it is an 'aggressive' option when used with the intention to provide user features. I just noted it recently, please, during the review comment this line. I will fix it for the next version.